emoji/src/main/java/com/gluonhq/emoji/DownloadbleEmojiSpriteLoader.java

```diff
@@ -0,0 +1,104 @@
+package com.gluonhq.emoji;
+
+import javafx.scene.image.Image;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.channels.Channels;
+import java.nio.channels.FileChannel;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.CompletableFuture;
+import java.util.logging.Logger;
+
+public class DownloadbleEmojiSpriteLoader implements EmojiSpriteLoader {
+
+    private static final Logger LOG = Logger.getLogger(DownloadbleEmojiSpriteLoader.class.getName());
+
+    private static final String COMMIT_NUMBER = "063f328d7951cb2e2a6678b06dcbdf8dd599fad7"; // tag 15.0.1
+    private static final String EMOJI_PNG_URL = "https://github.com/iamcal/emoji-data/blob/" + COMMIT_NUMBER + "/sheets-clean/sheet_apple_%s_clean.png?raw=true";
+
+    private static final String LOCAL_PATH = System.getProperty("user.home") + "/.gluon/emoji/" + COMMIT_NUMBER;
+    private static final int[] EMOJI_SIZES = new int[] { 20, 32, 64 };
+    private boolean initialized;
+
+    @Override
+    public boolean isInitialized() {
+        if (!initialized) {
+            for (int size : EMOJI_SIZES) {
+                String fileName = "sheet_apple_" + size + ".png";
+                if (!Files.exists(Paths.get(LOCAL_PATH, fileName))) {
+                    return false;
```

Comment:
add logging here, e.g. FINE in case the path exists, and INFO or even WARN in case the path does not exist. The case where the path does not exist should be rather exceptional, so we want to see this.

Reply:
The path will exists for the first time emojis are load in any application

Reply:
Yes, so that is only once in the total lifecycle of an application.

Reply:
added log if sprite file is not found